spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -544,7 +544,7 @@ def save_as_csv(agg_metric, fname_out, fname_in=None, append=False):
 
     # populate data
     with open(fname_out, 'a') as csvfile:
-        spamwriter = csv.writer(csvfile, delimiter=',')
+        spamwriter = csv.writer(csvfile, delimiter=',', quoting=csv.QUOTE_NONNUMERIC)
```

Comment:
That a useful option! I'll note that `delimiter=','` is the default already, but it doesn't hurt to be explicit.

Reply:
Ah, good to know! 

Honestly, this section could be cleaned up a little more (e.g. the fact that we use `spamwriter`, which I think is copied and pasted right out of a Python tutorial...) but I wanted to keep my changes minimal and easy to review.

I'm never quite sure when a good time is to clean up little things like this. :thinking: